### PR TITLE
docs(release): version 21.3.2 change log

### PR DESCRIPTION
**Pull request by huskylengcb**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,24 @@
...
+* **select:** empty style ([1a618ab](https://github.com/growingio/gio-design/commit/1a618ab8ca266ca4b985e43727b857620a85b4cb))
+* **select:** searchable => searchType ([#881](https://github.com/growingio/gio-design/issues/881)) ([ac32635](https://github.com/growingio/gio-design/commit/ac32635f5888798a41034c8272f7564f459cf0b9))
+* **tree-select:** 解决slide动画引发的bug ([#883](https://github.com/growingio/gio-design/issues/883)) ([09ab70d](https://github.com/growingio/gio-design/commit/09ab70d9daba53109a95402861b64e8c79c8da27))
+
+
+### BREAKING CHANGES
+
+* **select:** delete searchable and add searchType
+
+Co-authored-by: shiliqian <shiliqian@growingio.com>
```
Comment (jack0pan) [inline CHANGELOG.md:18]: 删除这两行

File: VERSION.md [hunk 2]
```diff
@@ -1,25 +1,19 @@
...
+* **select:** searchable => searchType ([#881](https://github.com/growingio/gio-design/issues/881)) ([ac32635](https://github.com/growingio/gio-design/commit/ac32635f5888798a41034c8272f7564f459cf0b9))
+* **tree-select:** 解决slide动画引发的bug ([#883](https://github.com/growingio/gio-design/issues/883)) ([09ab70d](https://github.com/growingio/gio-design/commit/09ab70d9daba53109a95402861b64e8c79c8da27))
 
 
 ### BREAKING CHANGES
 
-* **select:** searchable:boolean -> searchable:'custom' | 'inner'
+* **select:** delete searchable and add searchType
 
 Co-authored-by: shiliqian <shiliqian@growingio.com>
```
Comment (jack0pan) [inline VERSION.md:18]: 删除这两行

File: CHANGELOG.md (hunk 1 above)
Comment (huskylengcb) [reply CHANGELOG.md]: done

File: VERSION.md (hunk 2 above)
Comment (huskylengcb) [reply VERSION.md]: done

**Review by jack0pan — APPROVED** [review]

**Pull request merged**